docs/examples/wpi-many/wpi-many-custom-checker-example.sh

```diff
@@ -71,6 +94,6 @@ bash wpi-many.sh -o "${workingdir}/${checkername}-${repolistbase}" \
      -t ${timeout} \
      -- \
      --checker "${checker}" \
-     --quals "${qual_classpath}" \
-     --lib "${checker_classpath}" \
-     --stubs "${custom_stubs}"
+     ${quals_arg:+"--quals ${qual_classpath}"} \
```

Comment:
This quoting is too broad.  It should be
```
${quals_arg:+--quals "${qual_classpath}"}
```
(See the very last example at https://github.com/koalaman/shellcheck/wiki/SC2086.)

Reply:
Nice catch. I've fixed it.